core/controller/src/main/scala/whisk/core/controller/Authenticate.scala

```diff
@@ -79,4 +74,25 @@ trait Authenticate {
             Future.successful(None)
         }
     }
+
+  def validateCertificate(name: Option[String])(implicit transid: TransactionId): Future[Option[Identity]] = {
+      name flatMap { entityName =>
+          val name = entityName.substring("CN=".length, entityName.length)
+          Try {
+              //It also supports getting Identity by subject
+              val future = Identity.get(authStore, EntityName(name)) map { result =>
```

Comment:
For openwhisk model relation, subject : namespace == 1: n, namespace : auth key = 1: 1

* Use `namesapce value` to `CN` value, for openwhisk itself,  i think it is ok. 
* But for general meaning, the client certificate like `compressed username/password`, but for here, is `namespace`, feel strange.

What do you think about it?